### ISSUE TITLE
default to password only with autotype when no user is defined

### ISSUE THREAD

**Issue by benoliver999** [carnager/rofi-pass#113]
This might just apply to my workflow so I won't get hurt feelings if people don't like the idea:

I have quite a few files that just have the `pass` field. These are mainly for things where you don't enter a username e.g.  user passwords over ssh, api keys etc etc

It would save me a keystroke if the 'autotype' option defaults back to `pass` when no `user` field is found (or `default_autotype` cannot be resolved)

Right now I see the default is to either fall back to `whoami` or to `default_user`

**Comment by carnager** [comment]
why dont you just use Alt+2

**Comment by moviuro** [comment]
The empty string (`default_user=""`) would make more sense, I think.

**Comment by benoliver999** [comment]
Yeah, I guess when you drill down what I am really questioning is the value of `whoami` when no `default_user` is set. For me it's not that useful, but it doesn't bother me that much if people prefer it as it is.

**Comment by rattletat** [comment]
@carnager Any updates on this? I would love to see this since I use passwords of varying lengths and it is sometimes hard to estimate when rofi-pass finished typing a password. Or is there a configuration option for adding :enter after the password is typed?

**Comment by moviuro** [comment]
@rattletat probably not related to the current thread?

Try editing the config: https://github.com/carnager/rofi-pass/blob/136644167c06e842dd4f49541de1f9b1feb74c42/config.example#L60

**Comment by rattletat** [comment]
@moviuro But this is not triggered when using `insert_pass`, right?
